ang/civicase/activity/directives/activity-icon.directive.js

```diff
@@ -15,12 +15,12 @@
     /**
      * Link function for activityIcon directive
      *
-     * @param {object} scope
-     * @param {object} elem
-     * @param {object} attrs
+     * @param {object} scope scope
+     * @param {object} elem element
+     * @param {object} attrs attributes
```

Comment:
Not a fan of mandatory param description 🤷‍♂

Reply:
I think I kind of agree, having this particular rule is forcing us to write param description, even when its unnecessary. But again, disabling the rule means, we are stepping into the rabbit hole of enabling/disabling rules at will, not sure. @swastikpareek what do you think?

Reply:
just leave it for now. But I don't believe all param descriptions are helpful. When we create a directive we know what the attributes mean (yes, I can forget the order, but not what they mean). Adding a description to something passed by angular seems redundant. 

But, for all other cases it seems helpful. So if it's 90% helpful and 10% annoying, then lets leave it.